server/controllers/stock/functions/stock_report.function.js

```diff
@@ -0,0 +1,192 @@
+const _ = require('lodash');
+const moment = require('moment');
+
+const db = require('../../../lib/db');
+const core = require('../core');
+
+exports.getData = getData;
+
+// default report parameters
+const DEFAULT_PARAMS = {
+  csvKey : 'stock_report',
+  filename : 'REPORT.STOCK_REPORT.TITLE',
+  orientation : 'landscape',
+};
+
+/**
+ * @method getData
+ *
+ * @description
+ * This method builds the monthly stock report
+ *
+ * GET /stock/stock_report
+ */
+async function getData(reqQuery) {
```

Comment:
Hmm... A lot of this code looks identical to the code in `rumer.function.js`.  What exactly is different?  Could you maybe abstract out the common code into a common file that is used by both the `rumer` and this report?

https://github.com/Third-Culture-Software/bhima/blob/master/server/controllers/stock/functions/rumer.function.js

Reply:
The report has not been implemented, so the code must not be there. My bad